lib/LanguagePicker.js

```diff
@@ -67,7 +67,7 @@ LanguagePicker.prototype.newProcessor = function newProcessor() {
       }
     },
     getResult: () => {
-      const valueLangCodes = Object.keys(this.values);
+      let val;
```

Comment:
i meant don't declare it here either - because then the var has undefined value until assigned later.  The var declaration at the top is a leftover from the no `let/const` days (when it did make sense due to variable hoisting and related bugs) - see posts like [this](https://medium.com/@bluepnume/theres-no-need-to-define-all-javascript-vars-once-at-the-top-of-a-function-and-there-hasn-t-been-a66b31f21822).  So just move this line to where you assign it - it makes code more concise, and doesn't pollute the scope with vars until they are needed.

Reply:
LOL, silly me, I commented too fast I think :)